test/isContractDeployment.js

```diff
@@ -0,0 +1,379 @@
+    })
+
+    it('FEE_DELEGATED_SMART_CONTRACT_EXECUTION', () => {
+        const txObject = {
+            type: 'FEE_DELEGATED_SMART_CONTRACT_EXECUTION',
+            from: '0x90B3E9A3770481345A7F17f22f16D020Bccfd33e',
+            to: '0x5e008646fde91fb6eda7b1fdabc7d84649125cf5',
+            nonce: 12,
+            value: '0xa',
+            data: '0x6353586b00000000000000000000000090b3e9a3770481345a7f17f22f16d020bccfd33e',
+            gas: '0x3b9ac9ff',
+            gasPrice: '0x0',
+            chainId: '0x1',
+        }
+        expect(caver.utils.isContractDeployment(txObject)).to.false
+        expect(caver.utils.isContractDeployment(coverInitialForTest(txObject))).to.false
+    })
+
+    it('FEE_DELEGATED_SMART_CONTRACT_EXECUTION_WITH_RATIO', () => {
+        const txObject = {
+            type: 'FEE_DELEGATED_SMART_CONTRACT_EXECUTION_WITH_RATIO',
+            from: '0x90B3E9A3770481345A7F17f22f16D020Bccfd33e',
+            to: '0x5e008646fde91fb6eda7b1fdabc7d84649125cf5',
+            nonce: 13,
+            data: '0x6353586b00000000000000000000000090b3e9a3770481345a7f17f22f16d020bccfd33e',
+            gas: '0x3b9ac9ff',
+            gasPrice: '0x0',
+            chainId: '0x1',
+            feeRatio: 66,
+            value: '0xa',
+        }
+        expect(caver.utils.isContractDeployment(txObject)).to.false
+        expect(caver.utils.isContractDeployment(coverInitialForTest(txObject))).to.false
+    })
+
+    it('CANCEL', () => {
+        const txObject = {
+            type: 'CANCEL',
+            from: '0x90B3E9A3770481345A7F17f22f16D020Bccfd33e',
+            nonce: '0xe',
+            gasPrice: '0x19',
+            gas: '0x3b9ac9ff',
+            chainId: '0x1',
+        }
+        expect(caver.utils.isContractDeployment(txObject)).to.false
+        expect(caver.utils.isContractDeployment(coverInitialForTest(txObject))).to.false
+    })
+
+    it('FEE_DELEGATED_CANCEL', () => {
+        const txObject = {
+            type: 'FEE_DELEGATED_CANCEL',
+            from: '0x90B3E9A3770481345A7F17f22f16D020Bccfd33e',
+            nonce: 15,
+            gas: '0x3b9ac9ff',
+            gasPrice: '0x19',
+            chainId: '0x1',
+        }
+        expect(caver.utils.isContractDeployment(txObject)).to.false
+        expect(caver.utils.isContractDeployment(coverInitialForTest(txObject))).to.false
+    })
+
+    it('FEE_DELEGATED_CANCEL_WITH_RATIO', () => {
+        const txObject = {
+            type: 'FEE_DELEGATED_CANCEL_WITH_RATIO',
+            from: '0x90B3E9A3770481345A7F17f22f16D020Bccfd33e',
+            nonce: 16,
+            gas: '0x3b9ac9ff',
+            gasPrice: '0x0',
+            chainId: '0x1',
+            feeRatio: 88,
+        }
+        expect(caver.utils.isContractDeployment(txObject)).to.false
+        expect(caver.utils.isContractDeployment(coverInitialForTest(txObject))).to.false
+    })
+
+    it('CHAIN_DATA_ANCHROING', () => {
```

Comment:
```suggestion
    it('CHAIN_DATA_ANCHORING', () => {
```

Reply:
I think.... CHAIN_DATA_ANCHORING has to be modify chain transaction type string name..... ㅠㅠㅠ